cmd/export.go

```diff
@@ -43,41 +45,45 @@ func init() {
 		Args:  cobra.MaximumNArgs(1),
 
 		RunE: func(_ *cobra.Command, _ []string) error {
-			return export.Users(usersWPassword, exportFile)
+			return export.Users(usersWPassword, exportFile, pretty)
 		},
 	}
 
 	users.Flags().BoolVar(&usersWPassword, "withPassword", false, "Include each user's password")
 
 	cmd.AddCommand(users)
 
-	cmd.AddCommand(&cobra.Command{
+	var medias = &cobra.Command{
```

Comment:
Could you please revert the creation of vars for these commands since it's not necessary.